frame/staking/src/pallet/mod.rs

```diff
@@ -124,6 +125,18 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominations: Get<u32>;
 
+		/// `HistoryDepth` stores the number of previous eras for which
```

Comment:
I think this value is actually used for the first key of a number of double maps as well 🙈 

(also, same comment about comments being 100 in width, I used this https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)

Reply:
I could see this used only in Claimed Rewards in the Staking Ledger